python/morpheus/morpheus/_lib/cudf_helpers.pyx

```diff
@@ -29,6 +29,224 @@ from cudf._lib.column cimport Column
 from cudf._lib.utils cimport data_from_unique_ptr
 from cudf._lib.utils cimport table_view_from_table
 
+# isort: off
+
+# imports needed for get_element, which is required by from_column_view_with_fix
+cimport pylibcudf.libcudf.copying as cpp_copying
+from pylibcudf.libcudf.column.column_view cimport column_view
+from libcpp.memory cimport make_unique, unique_ptr
+from pylibcudf.libcudf.scalar.scalar cimport scalar
+from cudf._lib.scalar cimport DeviceScalar
+
+# imports needed for from_column_view_with_fix
+import rmm
+from libc.stdint cimport uintptr_t
+from cudf.core.buffer import (
+    # Buffer,
+    ExposureTrackedBuffer,
+    SpillableBuffer,
+    # acquire_spill_lock,
+    as_buffer,
+    # cuda_array_interface_wrapper,
+)
+cimport pylibcudf.libcudf.types as libcudf_types
+from cudf._lib.types cimport (
+    dtype_from_column_view,
+    # dtype_to_data_type,
+    # dtype_to_pylibcudf_type,
+)
+from cudf._lib.null_mask import bitmask_allocation_size_bytes
+
+# isort: on
+
+cdef get_element(column_view col_view, size_type index):
+
+    cdef unique_ptr[scalar] c_output
+    with nogil:
+        c_output = move(
+            cpp_copying.get_element(col_view, index)
+        )
+
+    return DeviceScalar.from_unique_ptr(
+        move(c_output), dtype=dtype_from_column_view(col_view)
+    )
+
+cdef Column from_column_view_with_fix(column_view cv, object owner):
```

Comment:
Please file an issue to remove this once upgraded to cuDF 24.12. And place a big fat comment surrounding this code, and linking to the issue, indicating why it is here.